src/cli_plugin/list/list.js

```diff
@@ -30,21 +30,15 @@
  * GitHub history for details.
  */
 
-import { statSync, readdirSync, readFileSync } from 'fs';
+import { statSync, readdirSync } from 'fs';
 import { join } from 'path';
 
 export function list(pluginDir, logger) {
   readdirSync(pluginDir).forEach((name) => {
     const stat = statSync(join(pluginDir, name));
 
     if (stat.isDirectory() && name[0] !== '.') {
-      try {
-        const packagePath = join(pluginDir, name, 'opensearch_dashboards.json');
-        const pkg = JSON.parse(readFileSync(packagePath, 'utf8'));
-        logger.log(pkg.id + '@' + pkg.version);
-      } catch (e) {
-        throw new Error('Unable to read opensearch_dashboards.json file for plugin ' + name);
-      }
```

Comment:
What's the reason to remove this?

Reply:
For the list command, the output printed was <pluginId>@<pluginVersion> but this same output could not be used in the remove command, this was mentioned in the same issue #322 in the latest comments, I removed this so that the actual folder name can be printed which the user can use to remove the plugin with the remove command. LMK if you think this could be better.